bin/update

```diff
@@ -13,5 +13,10 @@ end
 require gem_root.join("spec/manageiq/lib/manageiq/environment").to_s
 ManageIQ::Environment.manageiq_plugin_update(gem_root)
 
+unless ENV["SKIP_DATABASE_SETUP"]
```

Comment:
NOTE I pulled this from core's `bin/update` but it isn't clear when it is required and if we need it here.

Reply:
Yeah okay I'll drop this then, core also has an option for skipping the UI JS updates which isn't here